components/HomepageSlider/HomepageSlider.tsx

```diff
@@ -0,0 +1,105 @@
+import { useState, useEffect } from 'react';
+import { useOnWindowResize } from '../../custom_hooks/useOnWindowResize';
+import styles from "./HomepageSlider.module.scss";
+import Slide from './Slide';
+
+export default function HomepageSlider() {
+    const slide_amount = 2 // starts at 0
+    const [slide_size, setSlideSize] = useState(0);
+    const [slide_distance, setSlideDistance] = useState(0);
+    const [slide_position, setSlidePosition] = useState(0);
+    const [slide_card, setSlideCard] = useState(false);
+    const [mouse_down_start, setMouseDownStart] = useState(0);
+    const window_resize = useOnWindowResize();
+
+    const nextOrPrevSlide = (direction:string) => {
+        if (direction === "previous" && !(slide_position <= 0)) {
```

Comment:
I think its better to extract `previous` and `next` string to separate constants and use that constants in different places
``` const PREVIOUS = "previous" ```
and use it like ```if (direction===PREVIOUS ... ```
```nextOrPrevSlide(PREVIOUS)```

Reply:
Good point! Fixed that.